elyra/templates/pipeline/pipeline_properties_template.jinja2

```diff
@@ -0,0 +1,127 @@
+{
+  "current_parameters": {
+    "name": "",
+    "runtime": "",
+    "description": "",
+    "elyra_runtime_image": null,
+    "elyra_env_vars": []
+  },
+  "parameters": [
+    {
+      "id": "name"
+    },
+    {
+      "id": "runtime"
+    },
+    {
+      "id": "description"
+    },
+    {
+      "id": "elyra_runtime_image"
+    },
+    {
+      "id": "elyra_env_vars"
+    }
+  ],
+  "uihints": {
+    "id": "nodeProperties",
+    "parameter_info": [
+      {
+        "control": "custom",
+        "custom_control_id": "DisplayControl",
+        "parameter_ref": "name",
+        "label": { "default": "Pipeline Name" }
+      },
+      {
+        "control": "custom",
+        "custom_control_id": "DisplayControl",
+        "parameter_ref": "runtime",
+        "label": { "default": "Pipeline Runtime" }
+      },
+      {
+        "control": "custom",
+        "custom_control_id": "StringControl",
+        "parameter_ref": "description",
+        "label": { "default": "Pipeline Description" },
+        "data": {
+          "placeholder": "Pipeline description",
+          "format": "multiline"
+        }
+      },
+      {
+        "parameter_ref": "elyra_runtime_image",
+        "control": "custom",
+        "custom_control_id": "EnumControl",
+        "label": { "default": "Runtime Image" },
+        "description": {
+          "default": "Container image used as execution environment.",
+          "placement": "on_panel"
+        },
+        "data": {
+          "items": [],
+          "required": false
+        }
+      },
+      {
+        "parameter_ref": "elyra_env_vars",
+        "control": "custom",
+        "custom_control_id": "StringArrayControl",
+        "label": {
+          "default": "Environment Variables"
+        },
+        "description": {
+          "default": "Environment variables to be set on the execution environment.",
+          "placement": "on_panel"
+        },
+        "data": {
+          "placeholder": "env_var=VALUE"
+        }
+      }
+    ],
+    "group_info": [
+      {
+        "id": "nodeGroupInfo",
+        "type": "panels",
+        "group_info": [
+          {
+            "id": "name",
+            "type": "controls",
+            "parameter_refs": ["name"]
+          },
+          {
+            "id": "runtime",
+            "type": "controls",
+            "parameter_refs": ["runtime"]
+          },
+          {
+            "id": "description",
+            "type": "controls",
+            "parameter_refs": ["description"]
+          },
+          {
+            "id": "elyra_outputCategoryHeader",
+            "type": "textPanel",
+            "class_name": "elyra_categoryHeader",
+            "label": {
+              "default": "Generic nodes defaults"
+            },
+            "description": {
+              "default": "These values are applied to components ",
```

Comment:
@akchinSTC is there a reason you changed this text? I had set it to the exact text @ptitzler recommended previously in the comments.

Reply:
Will revert these changes, I accidentally included a commit when i squashed locally.